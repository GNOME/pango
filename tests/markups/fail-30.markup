<span size="blue">test</span>
