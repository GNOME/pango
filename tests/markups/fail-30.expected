ERROR: Value of 'size' attribute on <span> tag on line 1 could not be parsed; should be an integer, or a string such as 'small', not 'blue'